.circleci/cimodel/data/pytorch_build_data.py

```diff
@@ -39,14 +39,16 @@
                 # and
                 # https://github.com/pytorch/pytorch/blob/master/.jenkins/pytorch/build.sh#L153
                 # (from https://github.com/pytorch/pytorch/pull/17323#discussion_r259453144)
+                X("3.6"),
```

Comment:
We will lose libtorch test fon CUDA 9 both on PR and on master branch. Is that OK?

Reply:
Yeah, that seems fine, as long as libtorch is being tested on CUDA 10.1